### PR TITLE
[#2768] Merged messaging schema into repository (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: schemas/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## Schemas
```
Comment (korydraughn) [inline schemas/README.md:1]: Write README

Comment (korydraughn) [reply schemas/README.md:1]: Updated.

**Comment by korydraughn** [comment]
Do we need to carry forward the following files?
- https://github.com/irods/irods/tree/main/configuration_schemas/samples/
- https://github.com/irods/irods/tree/main/configuration_schemas/deploy_schemas_locally.py
- https://github.com/irods/irods/tree/main/configuration_schemas/README.md
- https://github.com/irods/irods/tree/main/configuration_schemas/validator.js
- https://github.com/irods/irods_schema_messaging/tree/24c73702c88e94c3b159dac97fe7a0640dfc209d/README.md

These files seem unnecessary.

**Comment by trel** [comment]
> These files seem unnecessary.

Agreed.   We have scripts/validate_json.py and the rest no longer are as relevent since we're shipping the config files *in* the repo (and the packages) now.

**Review by alanking — APPROVED** [review]
I like it

**Comment by korydraughn** [comment]
Will add the pounds once tests come back clean.

**Comment by korydraughn** [comment]
Core tests pass.

#'d

**Pull request merged**